1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -0,0 +1,15 @@
+export function fizzbuzz(value: number): string {
+    if (value === undefined) throw new Error("Input must be provided");
+
+    if (value < 1) throw new Error("Input must be no less than 1");
+
+    if (value > 100) throw new Error("Input must be no more than 100");
```

Comment:
[imho]
This is a matter of preference, but you could also say it like this to save a few lines :)

```js
if (number < 1 || number > 100) { 
  throw new Error("Input must be between 1 and 100"); 
}
```